modules/distribution/src/repository/resources/conf/templates/repository/conf/tomcat/catalina-server.xml.j2

```diff
@@ -0,0 +1,73 @@
+
+    <Service className="org.wso2.carbon.tomcat.ext.service.ExtendedStandardService" name="Catalina">
+
+        <!--
+        optional attributes:
+
+        proxyPort="80"
+        -->
+        <Connector protocol="org.apache.coyote.http11.Http11NioProtocol"
+        {% for property_name,property_value in transport.http.items() %}
+                   {{property_name}}="{{property_value}}"
+                           {% endfor %}
+        />
+
+        <!--
+        optional attributes:
+
+        proxyPort="443"
+        Added sslEnabledProtocols="TLSv1,TLSv1.1,TLSv1.2" for poodle vulnerability fix
+        -->
+        <Connector protocol="org.apache.coyote.http11.Http11NioProtocol"
+        {% for property_name,property_value in transport.https.items() %}
+                   {{property_name}}="{{property_value}}"
+                   {% endfor %}
+        />
+
+        <Engine name="Catalina" defaultHost="localhost">
+
+            <!--Realm className="org.apache.catalina.realm.MemoryRealm" pathname="${carbon.home}/repository/conf/tomcat/tomcat-users.xml"/-->
+
+            <Realm className="org.wso2.carbon.tomcat.ext.realms.CarbonTomcatRealm"/>
+
+            <Host name="localhost" unpackWARs="true" deployOnStartup="false" autoDeploy="false"
+                  appBase="${carbon.home}/repository/deployment/server/webapps/">
+                <Valve className="org.wso2.carbon.tomcat.ext.valves.RequestEncodingValve" encoding="UTF-8"/>
+                <Valve className="org.wso2.carbon.tomcat.ext.valves.RequestCorrelationIdValve"
+                       headerToCorrelationIdMapping="{'activityid':'Correlation-ID'}" queryToCorrelationIdMapping="{'RelayState':'Correlation-ID'}"/>
+                <Valve className="org.wso2.carbon.tomcat.ext.valves.CarbonContextCreatorValve"/>
+                <Valve className="org.apache.catalina.valves.AccessLogValve" directory="${carbon.home}/repository/logs"
+                       prefix="http_access_" suffix=".log" pattern="%h %l %u %t %r %s %b %{Referer}i %{User-Agent}i %T"/>
+                <Valve className="org.wso2.carbon.tomcat.ext.valves.CarbonStuckThreadDetectionValve" threshold="600"/>
+                <Valve className="org.wso2.carbon.tomcat.ext.valves.CompositeValve"/>
+
+                <!-- Authentication and Authorization valve for the rest apis and we can configure context for this in identity.xml.j2  -->
+                <Valve className="org.wso2.carbon.identity.auth.valve.AuthenticationValve"/>
+                <Valve className="org.wso2.carbon.identity.authz.valve.AuthorizationValve"/>
+                <Valve className="org.wso2.carbon.identity.context.rewrite.valve.TenantContextRewriteValve"/>
+                <!--Error pages -->
+                <Valve className="org.apache.catalina.valves.ErrorReportValve" showServerInfo="false" showReport="false"/>
+            </Host>
+        </Engine>
+    </Service>
+</Server>
+
```

Comment:
Remove additional newline

Reply:
Fixed - https://github.com/wso2/product-is/pull/7059